Dockerfile

```diff
@@ -16,19 +16,12 @@ ADD get-language /go/src/github.com/openshift/odo-supervisord-image/get-language
 WORKDIR /go/src/github.com/openshift/odo-supervisord-image/get-language
 RUN go build -o /tmp/getlanguage  getlanguage.go
 
-
-# Build dumb-init
-FROM registry.access.redhat.com/ubi7/ubi AS dumbinitbuilder
-WORKDIR /tmp/dumb-init-src
-RUN yum -y install gcc make binutils
-COPY vendor/dumb-init /tmp/dumb-init-src
-RUN gcc -std=gnu99 -s -Wall -Werror -O3 -o dumb-init dumb-init.c
-
+ADD go-init/main.go /go/src/go-init.go
```

Comment:
it would be cleaner to add the whole go-init directory as `/go/src/github.com/pablo-ruth/go-init`. The same way we do it with supervisord and get-language

`ADD go-init  /go/src/github.com/pablo-ruth/go-init`

Reply:
ok, will update